backend/backend/urls.py

```diff
@@ -79,7 +80,8 @@
 	# Stats Views
 	path('users/<int:user_id>/stats/', StatsView.as_view()),
 	path('leaderboard/', LeaderboardListView.as_view()),
-	path('users/<int:user_id>/graphs/', StatsGraphView.as_view())
+	path('users/<int:user_id>/graphs/', StatsGraphView.as_view()),
```

Comment:
Technically, neither a bar nor a pie chart are graphs

Reply:
should I rename them to chart?

Reply:
maybe just something like users/user_id/last_played_matches_score?